test/integration/standup.test.js

```diff
@@ -0,0 +1,76 @@
+const chai = require("chai");
+const app = require("../../server");
+const addUser = require("../utils/addUser");
+const userData = require("../fixtures/user/user")();
+const authService = require("../../services/authService");
+const cleanDb = require("../utils/cleanDb");
+const { object } = require("joi");
```

Comment:
I think its better to separate library import from local imports

Reply:
sure i will include it in subsequent PRs. as I have to write tests for 2 more apis

Reply:
done